sktime/transformations/series/func_transform.py

```diff
@@ -18,12 +18,12 @@ class FunctionTransformer(BaseTransformer):
     r"""Constructs a transformer from an arbitrary callable.
 
     A FunctionTransformer forwards its y (and optionally X) arguments to a
-    user-defined function or function object and returns the result of this
+    user-defined function (or callable object) and returns the result of this
```

Comment:
small enough to have it hide in another PR, though my general recommendation is to make separate PR for sth like this, it is one that would be instant-accepted and merged. The danger is that insta-merge ride-along changes get stalled with the host PR and clutter its review.
(you also hurt your honest contribution statistics, if that were something you care about)

Reply:
Good point! Moved these changes to #5634